### ISSUE TITLE
Designer Tab - Upload and Clear Logo buttons - Buttons must have discernible text

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-creator#7785]
In v2, this issue doesn't occur - these elements are span elements.

In v3, these elements are buttons and no longer inputs. 

[View Demo](https://plnkr.co/edit/g7ZoPLYJPWCrwSWd)

<img width="2560" height="1229" alt="Image" src="https://github.com/user-attachments/assets/9bff3784-b991-4863-93e1-f47e9aef3b49" />

The AXE checker shows "Buttons must have discernible text" for the Upload and Clear logo buttons:

Element does not have inner text that is visible to screen readers.

<img width="2560" height="1229" alt="Image" src="https://github.com/user-attachments/assets/397b51c2-ec2e-44af-8429-0541fbf67f17" />
